tests/k6/tests/scripts/generate_tokens.sh

```
@@ -0,0 +1,92 @@
+    fi
+    echo "ssn,resource,scopes,token" > $enduser_tokenfile
+    while IFS=, read -r ssn resource scopes
+    do
+        url="https://altinn-testtools-token-generator.azurewebsites.net/api/GetPersonalToken?env=$env&scopes=$scopes&pid=$ssn&ttl=3600"
+        token=$(curl -s -f $url -u "$tokengenuser:$tokengenpasswd" )
+        if [ $? -ne 0 ]; then
+            echo "Error: Failed to generate personal token for: $ssn, $scopes "
+            continue
+        fi
+        echo "$ssn,$resource,$scopes,$token" >> $enduser_tokenfile
+        if [ $? -ne 0 ]; then
```

Shellcheck

[warning] This $? refers to echo/printf, not a previous command. Assign to variable to avoid it being overwritten.

(SC2320)